client/my-sites/checkout/composite-checkout/composite-checkout.tsx

```diff
@@ -578,8 +626,8 @@ export default function CompositeCheckout( {
 						<CheckoutStepAreaWrapper>
 							<EmptyCart />
 							<SubmitButtonWrapper>
-								<Button buttonType="primary" fullWidth onClick={ goToPlans }>
-									{ translate( 'Browse our plans' ) }
+								<Button buttonType="primary" fullWidth onClick={ goToPreviousPage }>
+									{ translate( 'Return to previous page' ) }
```

Comment:
Minor but I wonder if wording like "Go back" might read a little better? If for example you came here from a direct link, then there is no "previous" page, whereas "go back" is a little more generic sounding, I think?

Reply:
I like this wording better, looks more correct from my testing as well